prototype_source/pt2e_quant_ptq_static_x86_inductor.rst

```diff
@@ -0,0 +1,187 @@
+        # use the API of `capture_pre_autograd_graph`
+        # Note 1: `capture_pre_autograd_graph` is also a short-term API, it will be updated to use the official `torch.export` API when that is ready.
+        exported_model = capture_pre_autograd_graph(
+            model,
+            example_inputs
+        )
+        # Note 2: if you are using the PyTorch 2.1 release binary or building from source with the PyTorch 2.1 release branch,
+        # please use the API of `torch._dynamo.export` to capture the FX Graph.
+        # exported_model, guards = torch._dynamo.export(
+        #     model,
+        #     *copy.deepcopy(example_inputs),
+        #     aten_graph=True,
+        # )
+
+
+Next, we will have the FX Module to be quantized.
+
+2. Apply Quantization
+----------------------------
+
+After we capture the FX Module to be quantized, we will import the Backend Quantizer for X86 CPU and configure how to
+quantize the model.
+
+::
+
+    quantizer = X86InductorQuantizer()
+    quantizer.set_global(xiq.get_default_x86_inductor_quantization_config())
+
+.. note::
+
+   The default quantization configuration in ``X86InductorQuantizer`` uses 8-bits for both activations and weights.
+  When Vector Neural Network Instruction is not available, the oneDNN backend silently chooses kernels that assume
+  `multiplications are 7-bit x 8-bit <https://oneapi-src.github.io/oneDNN/dev_guide_int8_computations.html#inputs-of-mixed-type-u8-and-s8>`_. In other words, potential
+  numeric saturation and accuracy issue may happen when running on CPU without Vector Neural Network Instruction.
+
+After we import the backend-specific Quantizer, we will prepare the model for post-training static quantization.
+``prepare_pt2e`` folds BatchNorm operators into preceding Conv2d operators, and inserts observers in appropriate places in the model.
+
+::
+
+    prepared_model = prepare_pt2e(exported_model, quantizer)
+
+Now, we will calibrate the ``prepared_model`` after the observers are inserted in the model.
+
+::
+
+    # We use the dummy data as an example here
+    prepared_model(*example_inputs)
+
+    # Alternatively: user can define the dataset to calibrate
+    # def calibrate(model, data_loader):
+    #     model.eval()
+    #     with torch.no_grad():
+    #         for image, target in data_loader:
+    #             model(image)
+    # calibrate(prepared_model, data_loader_test)  # run calibration on sample data
+
+Finally, we will convert the calibrated Model to a quantized Model. ``convert_pt2e`` takes a calibrated model and produces a reference quantized model.
+
+::
+
+    converted_model = convert_pt2e(prepared_model)
+
+After these steps, we finished running the quantization flow and we will get the reference quantized model.
+
+
+3. Lower into Inductor
+------------------------
+
+After we get the reference quantized model, we will further lower it to the inductor backend.
```

Comment:
please remove reference here as well (we redefined reference quantized model to mean the model with integer representations in pt2e flow..)

Reply:
Ok, Got it. I think all the `reference` has been removed.